src/models/searchservice/response/Result.ts

```diff
@@ -35,8 +35,8 @@ export interface Result {
    * distanceFromFilter value will be from the search result to 'New York'.
    */
   distanceFromFilter?: number;
-  /** An array of {@link HighlightedValue | highlighted values} associated with the result. */
-  highlightedValues?: HighlightedValue[];
+  /** A tree of {@link HighlightedFields | highlighted fields} emphasized by the api. */
```

Comment:
I think we should leave out the word "tree" here. At the top-level, the `highlightedFields` is an object, keyed by fieldId. The phrase "tree" may lead someone to believe that there's a single, root node in `highlightedFields`, when in fact there may be many top-level entries.

Reply:
Hmm I feel like this is still a tree, since the single root node is `highlightedFields`, but I think if somebody wants to know about the structure they can probably just click the link